cmd/pFaTools/extract.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta/pFasta"
+	"github.com/vertgenlab/gonomics/numbers/parse"
+	"log"
+	"os"
+)
+
+// ExtractSettings defines the usage settings for the pFa extract subcommand.
+type ExtractSettings struct {
+	InFile  string
+	Chrom   string
+	OutName string
+	OutFile string
+	Start   int
+	End     int
+}
+
+// extractUsage defines the usage statement for the pFaTools extract subcommand.
+func extractUsage(extractFlags *flag.FlagSet) {
+	fmt.Printf("pFaTools extract - a tool for extracting a subsequence from a pFa.\n" +
+		"A note on input and output file types:\n" +
+		"\tpFa (probabilistic FASTA) encodes position-wise probabilities in matrix entries. Columns should therefore sum to 1.\n" +
+		"Usage:\n" +
+		"pFaTools extract in.pFa chrom out.fasta start end\n" +
```

Comment:
Output of extract is a .pfa, not a .fasta

Reply:
Changed